crates/block-producer/src/store/mod.rs

```diff
@@ -61,6 +76,8 @@ pub struct TransactionInputs {
     /// Maps each consumed notes' nullifier to block number, where the note is consumed
     /// (`zero` means, that note isn't consumed yet)
     pub nullifiers: BTreeMap<Nullifier, u32>,
+    /// List of notes that were not found in the store
+    pub missing_notes: Vec<NoteId>,
```

Comment:
Let's rename this to `missing_unauthenticated_notes` and update the comment accordingly.